### DISCUSSION TITLE
Array key param

### DISCUSSION THREAD

**Discussion by tmlyczynski** [pbeshai/use-query-params#246] [Q&A]
Hi, i try to change array encoding type right now function produce for example "192.168.0.1:8080/?test[]=1&test[]=2" but i try to get rid of those [] sings in url, any ideas how to do it?

**Comment by pbeshai** [comment]
Hi there, this is likely being controlled by whatever you chose to use for your `objectToSearchString` function. For instance, if you're using [query-string](https://github.com/sindresorhus/query-string) and you overrode the default to specify you wanted "bracket" style ([see docs](https://github.com/sindresorhus/query-string#arrayformat)), it would do what you're sharing, but the default does not add those brackets (either in query-string or the URLSearchParams)
